uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -9,7 +9,7 @@ <h4 class="modal-title">New Document</h4>
   <div class="modal-body" nv-file-drop uploader="uploader">
     <div class="container">
       <!-- Search and select by name and email -->
-      <div class="row">
+      <div class="row" ng-hide="role === 'staff'">
```

Comment:
you have this logic duplicated everywhere in the view. Encapsulate it in an helper method that you publish on the scope, something like
```html
<div class="row" ng-hide="isRole('staff')">
```